backend/app/transcendence/.eslintrc.js

```diff
@@ -0,0 +1,327 @@
+module.exports = {
```

Comment:
{ on end of line.

Reply:
Plain object Affectation must follow this style. (when '=' followed by a '{' or ']'), other case will be at a new line, object definition, function.